### PR TITLE
Extract instrumentation package

### LINKED ISSUE #75: Extract subcomponents
Extract subcomponents into monorepo module with own packages.

- [x] instrumentation (#119)
- [x] CFG generator (#103)

### PR DESCRIPTION
**Pull request by dstallenberg**
Closes #75

### CODE REVIEW

File: packages/instrumentation-javascript/NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SynTest Javascript
```
Comment (mitchellolsthoorn) [inline packages/instrumentation-javascript/NOTICE:1]:
```suggestion
SynTest Framework - SynTest JavaScript
```

File: packages/instrumentation-javascript/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Abstract Syntax Tree Visitor library for JavaScript
```
Comment (mitchellolsthoorn) [inline packages/instrumentation-javascript/README.md:1]: Should be updated

File: packages/instrumentation-javascript/package.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@syntest/instrumentation-javascript",
+  "version": "0.0.1",
```
Comment (mitchellolsthoorn) [inline packages/instrumentation-javascript/package.json:3]:
```suggestion
  "version": "0.1.0",
```

File: packages/javascript/package.json [hunk 4]
```diff
@@ -49,9 +49,10 @@
     "@babel/preset-env": "7.19.1",
     "@babel/register": "7.18.9",
     "@istanbuljs/schema": "0.1.3",
-    "@syntest/cfg-javascript": "^0.0.2-beta.0",
     "@syntest/ast-javascript": "*",
+    "@syntest/cfg-javascript": "^0.0.2-beta.0",
     "@syntest/core": "^0.3.1-beta.0",
+    "@syntest/instrumentation-javascript": "^0.0.1",
```
Comment (mitchellolsthoorn) [inline packages/javascript/package.json:55]: Shouldn't these be `*`?

File: packages/instrumentation-javascript/NOTICE (hunk 1 above)
Comment (dstallenberg) [reply packages/instrumentation-javascript/NOTICE:1]: Fixed.

File: packages/instrumentation-javascript/README.md (hunk 2 above)
Comment (dstallenberg) [reply packages/instrumentation-javascript/README.md:1]: Fixed.

File: packages/instrumentation-javascript/package.json (hunk 3 above)
Comment (dstallenberg) [reply packages/instrumentation-javascript/package.json:3]: Should be fixed automatically.

File: packages/javascript/package.json (hunk 4 above)
Comment (dstallenberg) [reply packages/javascript/package.json:55]: Fixed.

**Review by mitchellolsthoorn — DISMISSED** [review]

**Pull request merged**